db/fixtures/miq_product_features.yml

```diff
@@ -6839,6 +6839,15 @@
       :description: View Metrics
       :feature_type: view
       :identifier: metrics_view
+  - :name: Metric Rollups
+    :description: Everything under Metric Rollups
+    :feature_type: node
+    :identifier: metric_rollups
+    :children:
+    - :name: View
+      :description: View Metric Rollups
+      :feature_type: view
+      :identifier: metric_rollups_view
```

Comment:
NOTE the `/api/metric_rollups` API used the `metrics` and `metrics_view` features which would collide with `/api/metrics`